pisces/utils.py

```diff
@@ -220,20 +220,16 @@ def max2epochs(data, fs, epoch):
     return time_counts, counts
 
 # %% ../nbs/00_utils.ipynb 11
-from agcounts.extract import get_counts
+# from agcounts.extract import get_counts
 
-def build_ActiGraph_official(time_xyz, axis: int = 3) -> Tuple[np.ndarray, np.ndarray]:
-    freq = 50
-    counts = get_counts(time_xyz[:, 1:], freq=freq, epoch=15)[:, axis - 1]
-    times = np.linspace(time_xyz[0, 0], time_xyz[-1, 0], len(counts))
+# def build_ActiGraph_official(time_xyz, axis: int = 3) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
I actually [removed ag counts entirely](https://github.com/Arcascope/pisces/pull/9/commits/a513caee45c82cf11c8143de6a574af81caffeab) on my branch. So I'd say let's just cut it.